HHS/Head-Start-TTADP
# [TTAHUB-3444] Prevent clicking out of drawers

## Description of change
- Drawers should capture and trap focus (you can't tab out of them)

## How to test
Confirm that the drawers appear the same, don't break down at various screen sizes, and behave as described in the above "description of change" section.

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-3444


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
